pkg/cmd/run/view/view.go

```diff
@@ -658,24 +668,28 @@ func truncateAsUTF16(str string, max int) string {
 //     where the ID can apparently be negative.
 func attachRunLog(rlz *zip.Reader, jobs []shared.Job) {
 	for i, job := range jobs {
-		re := jobLogFilenameRegexp(job)
-		for _, file := range rlz.File {
-			if re.MatchString(file.Name) {
-				jobs[i].Log = file
-				break
-			}
+		// the normal job run log file is preferred over the legacy one. So, we
```

Comment:
Given https://github.com/cli/cli/pull/10769#issuecomment-2796956212

I'd suggest updating this comment to the effect of:

```go
// As a highest priority, we try to use the step logs first. We have seen zips that surprisingly contain
// step logs, normal job logs and legacy job logs. In this case, both job logs would be ignored. We have
// never seen a zip containing both job logs and no step logs, however, it may be possible. In that case
// let's prioritise the normal log over the legacy one.
```

Reply:
Done.